### PR TITLE
chore: added all files to the store build & aligned code with the Mender

### PR DESCRIPTION
**Pull request by aleksandrychev**

### CODE REVIEW

File: packages/store/onboardingSlice/thunks.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 // @ts-nocheck
-import { DEVICE_STATES, onboardingSteps as onboardingStepNames, orderedOnboardingSteps as onboardingSteps } from '@northern.tech/store/constants';
+import { DEVICE_STATES, onboardingSteps as onboardingStepNames } from '@northern.tech/store/constants';
 import { getOnboardingState as getCurrentOnboardingState, getUserCapabilities } from '@northern.tech/store/selectors';
```
Comment (mzedel) [inline packages/store/onboardingSlice/thunks.ts:16]: the `orderedOnboardingSteps` are needed further down

File: packages/store/onboardingSlice/constants.ts [hunk 2]
```diff
@@ -31,23 +31,3 @@ export const onboardingSteps = {
   DEPLOYMENTS_PAST_COMPLETED_FAILURE: 'deployments-past-completed-failure',
   ONBOARDING_CANCELED: 'onboarding-canceled'
 };
-
-export const orderedOnboardingSteps = [
```
Comment (mzedel) [inline packages/store/onboardingSlice/constants.ts:35]: these are required in the `thunks` for the onboardingSlice

File: packages/store/organizationSlice/constants.ts [hunk 3]
```diff
@@ -12,9 +12,10 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 // @ts-nocheck
-import { Integration } from '@northern.tech/store/api/types/MenderTypes';
 import { EXTERNAL_PROVIDER, apiUrl, useradmApiUrl } from '@northern.tech/store/constants';
 
+import { Integration } from '../api/types/Integration';
+
 export const auditLogsApiUrl = `${apiUrl.v1}/auditlogs`;
```
Comment (mzedel) [inline packages/store/organizationSlice/constants.ts:19]: please revert this to the `MenderTypes` reference

File: packages/store/organizationSlice/thunks.ts [hunk 4]
```diff
@@ -35,6 +34,7 @@ import hashString from 'md5';
 import Cookies from 'universal-cookie';
 
 import { actions, sliceName } from '.';
+import { Tenant } from '../../components/tenants/types';
 import { SSO_TYPES, auditLogsApiUrl, ssoIdpApiUrlv1, tenantadmApiUrlv1, tenantadmApiUrlv2 } from './constants';
```
Comment (mzedel) [inline packages/store/organizationSlice/thunks.ts:38]: please revert this to use the `Tenant` type from `MenderTypes`

File: packages/store/package.json [hunk 5]
```diff
@@ -34,6 +53,8 @@
     "dayjs": "^1.x",
     "jwt-decode": "^4.x",
     "md5": "^2.x",
+    "@emotion/react": "^11.5.0",
+    "@emotion/styled": "^11.3.0",
```
Comment (mzedel) [inline packages/store/package.json:57]: please align the version references with the other peerDependency versions

File: packages/store/tsconfig.json [hunk 6]
```diff
@@ -2,6 +2,10 @@
   "extends": "@northern.tech/typescript-config/react-app.json",
   "exclude": ["dist", "build", "node_modules", "**/*.test.js"],
   "compilerOptions": {
+  
+      "skipLibCheck": true,
+      "noEmitOnError": false,
+ 
     "rootDir": ".",
```
Comment (mzedel) [inline packages/store/tsconfig.json:9]:
```suggestion
  "compilerOptions": {
    "noEmitOnError": false,
    "rootDir": ".",
```
`skipLibCheck` is already defined via `react-app` => `base` + the formatting should be aligned

**Review by mzedel — CHANGES_REQUESTED** [review]
Thanks for the alignment, I hope we can soon rely more on the package publishing to make the switch to define the store here and no longer rely on these manual syncs.. 🤞

File: packages/store/tsconfig.json (hunk 6 above)
Comment (aleksandrychev) [reply packages/store/tsconfig.json:9]: leftovers from the previous tries... I will remove them

**Review by mzedel — APPROVED** [review]

**Pull request merged**